geotrek/feedback/fixtures/basic.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "model": "feedback.reportactivity",
+        "pk": 18,

```

Comment:
I'm not sure but with postgresql serial auto increment, I think use 18 as first value is not a good idea. Use 1 instead ?